Hangfire.Redis.StackExchange/RedisMonitoringApi.cs

```diff
@@ -638,7 +638,7 @@ public StatisticsDto GetStatistics()
                 {
                     var queueName = queue;
                     tasks[i] = pipeline.ListLengthAsync(_storage.GetRedisKey($"queue:{queueName}"), CommandFlags.HighPriority)
-						.ContinueWith(x => stats.Enqueued += x.Result);
+						.ContinueWith(x => { lock (stats) { stats.Enqueued += x.Result; } });
```

Comment:
Totally right!

Reply:
@marcoCasamento `ScheduledJobs` concurrently adding values to `jobs` / `states` dictionaries actually was much more of an issue :)